scss/components/button.scss

```diff
@@ -230,6 +235,8 @@ a.#{$block} {
       --fd-button-color: var(--fd-color-action-1);
       @include fd-var-color("color", fd-color("action", 1), --fd-button-color);
       @include fd-var-color("background-color", fd-color("neutral", 1), --fd-button-background-color);
+      box-shadow: 0 0 0 1px var(--fd-color-action-hover,#085CAF);
+      z-index: 1;
```

Comment:
I wouldn't have expected all these changes in the `button.scss` file for a DatePicker / InputGroup button issue.  Are these truly needed and if so, should they be part a PR to change or refactor buttons?